Added sysmail table from the PR however service broker tables are not…

moraja opened the pull request:
Added sysmail table from the PR however service broker tables are not in the PR and I didn't add them

PiJoCoder reviewed (changes requested):
Hey @moraja thank you for working on this. Overall looks good. There are a couple of things:
1. One of the rowsets isn't getting imported : sysmail_mailitems and its identifier is "-- sysmail_mailitems --". Can you fix that?
2. Could you use a consistent identation for the xml tags as follows: - `<Rowset>` is indented at 4 spaces, `<KnownColumns>` is indented at 6 spaces and `<Column>` is indented at 8 spaces. Add a blank line between each rowset. I had not warned you about this detail - sorry. In the past we had the rowsets in such various indentations that it was impossible to figure out for some what the parent/child relationship was. So we ended up establishing a strict formatting. I have also documented it now. I updated one rowset to show illustrate (committed and pushed).
3. Can you delete the comment at the bottom of the file as it is not needed.

moraja commented:
@PiJoCoder  I updated the indentation and sysmail mailitems, this one was there as sysmail_items becuse my repo for logscout was old, I updated it locally to generate the files as expected.
